healthcareai/develop_supervised_model.py

```diff
@@ -466,7 +467,7 @@ def advanced_random_forest_classifier(self, trees=200, scoring_metric='roc_auc',
 
         if randomized_search:
             if not hyperparameter_grid:
-                max_features = model_eval.calculate_rfmtry(len(self.X_test.columns), self.model_type)
+                max_features = healthcareai.common.helpers.calculate_random_forest_mtry_hyperparameter(len(self.X_test.columns), self.model_type)
```

Comment:
You're calculating this twice, potentially.  Can you do this once instead?

`max_features = rf_helper(len(self.X_test.columns), self.model_type)
if randomized_search:
  if not hyperparameter_grid:
    hyperparameter_grid = {'n_estimators': [10, 50, 200], 'max_features': max_features}
...
params = {'max_features': max_features}